cluster/packages.sls

```diff
@@ -1,11 +1,13 @@
 #required packages to install HA cluster
 
+{%- from "cluster/map.jinja" import cluster with context -%}
+
 {% set pattern_available = 1 %}
 {% if grains['os_family'] == 'Suse' %}
 {% set pattern_available = salt['cmd.retcode']('zypper search patterns-ha-ha_sles') %}
 {% endif %}
 
-{% if pattern_available == 0 %}
+{% if pattern_available == 0 and cluster.ha_repo is not defined %}
```

Comment:
@ayoub-belarbi This makes it restricted to our reality (Devel on OBS vs Official Repos). I would say we should have here a generic way to inform a specific repo, or my preference is to adjust the `ha-sap-terraform-deployment` project to allow dynamic repo priorities settings, so you can control the this outside of this package. I think the mixing Repo management inside this formula is wrong, as it is usually controled by SUMA, or generic formulas for all the servers, for example.